Backend/TemporalDataManager.py

```diff
@@ -1,18 +1,18 @@
 from datetime import datetime, timedelta
 import threading
 import time
+import DataStructures
 
 class TemporalDataManager:
 
     # TODO: set up instance of algorithm manager (and anything else needed)
     # TODO: tasked to set up threads that can access trends data retrieving functions
-    def __init__(self,  algomanager, loc, t=3):
+    def __init__(self,  algomanager):
         self.algo = algomanager
-        self.timer = t
```

Comment:
why remove the timer interval field?

Reply:
it was not used so i removed. I'll go implement the timer argument into the temporal thread timing now.